Formula/podman.rb

```diff
@@ -17,24 +17,45 @@ class Podman < Formula
   depends_on "go-md2man" => :build
   depends_on "qemu" if Hardware::CPU.intel?
 
+  resource "gvproxy" do
+    url "https://github.com/containers/gvisor-tap-vsock/archive/v0.1.0.tar.gz"
+    sha256 "e1e1bec2fc42039da1ae68d382d4560a27c04bbe2aae535837294dd6773e88e0"
+  end
+
   def install
-    system "make", "podman-remote-darwin"
-    bin.install "bin/darwin/podman"
+    os = "darwin"
+    on_linux do
+      os = "linux"
+    end
+
+    system "make", "podman-remote-#{os}"
+    on_macos do
+      bin.install "bin/#{os}/podman" => "podman-remote"
+      bin.install_symlink bin/"podman-remote" => "podman"
+    end
+    on_linux do
+      bin.install "bin/podman-remote"
```

Comment:
Not blocking this PR but for a future release it would be nice to have an upstream `make install` which encapsulates this logic.